### PR TITLE
Update Styling of the Pages

### PR DESCRIPTION
**Pull request by TMazenge**
I moved the enrolling wizard Javascript to another file.

### CODE REVIEW

File: src/main/webapp/js/script.js [hunk 1]
```diff
@@ -135,21 +93,19 @@ async function getReleaseSDKs(platform, releaseName) {
   const response = await fetch(`v1/platforms/${platform}/releases/${releaseName}/sdks`);
   const sdks = await response.json();
 
-  const listSDKs = sdks[releaseName];
+  console.log(sdks);
```
Comment (ankitaj224) [inline src/main/webapp/js/script.js:97]: Add a TODO to remove it later.

File: src/main/webapp/js/enroll.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+
+$(document).ready(function () {
```
Comment (ankitaj224) [inline src/main/webapp/js/enroll.js:2]: Detailed comments in this file would be helpful.

**Review by ankitaj224 — APPROVED** [review]

File: src/main/webapp/js/script.js (hunk 1 above)
Comment (TMazenge) [reply src/main/webapp/js/script.js:97]: Done!

File: src/main/webapp/js/enroll.js (hunk 2 above)
Comment (TMazenge) [reply src/main/webapp/js/enroll.js:2]: Added

**Pull request merged**